pkg/cmd/initialize/sigs.go

```diff
@@ -44,6 +45,8 @@ func CreateEventsFromSignatures(startId events.ID, sigs []detect.Signature) map[
 			evtDependency = append(evtDependency, eventDefID)
 		}
 
+		tags := set.New[string](append([]string{"signatures", "default"}, m.Tags...)...)
```

Comment:
nit: pretty sure you can do
` set.New[string]("signatures", "default", m.Tags...)

Reply:
I'll try again, but it complained about arguments string,string,[]string

Reply:
You're right jsut checked myself. odd that golang wouldn't support it, it's rather intuitive...